### ISSUE TITLE
Realme 7 Pro Camera Issue

### ISSUE THREAD

**Issue by egayghost** [phhusson/treble_experimentations#1836]

**Comment by phhusson** [comment]
What is the issue?

**Comment by egayghost** [comment]
It doesn't work

**Comment by phhusson** [comment]
What does "doesn't work" mean?
The camera app is not listed at all?

**Comment by egayghost** [comment]
> What does "doesn't work" mean?
> The camera app is not listed at all?

Camera app crashing

**Comment by phhusson** [comment]
Can you manually grant camera app camera permission from settings?

**Comment by egayghost** [comment]
> Can you manually grant camera app camera permission from settings?

No, it was only reported by one of your tester who tested the touchscreen fix for realme 7 pro, I'm the 2nd guy who make the tickets, but I will check your latest build maybe this end of the week.

**Comment by egayghost** [comment]
> Can you manually grant camera app camera permission from settings?

I just  tested it out, and yes permissions are all enabled but camera app is crashing

**Comment by egayghost** [comment]
[Log_2021-05-22_14-12-21.txt](https://github.com/phhusson/treble_experimentations/files/6525916/Log_2021-05-22_14-12-21.txt)
